src/controllers/_authentication_controllers.py

```diff
@@ -0,0 +1,31 @@
+from flask import request
+from src.config.soap_client import soap_client
+
+
+def login_handler():
+    try:
+        # Get JSON data from the request
+        data = request.json
+
+        if not data:
+            return {"message": "No JSON data provided in the request"}, 400
+
+        username = data.get("username")
+        password = data.get("password")
+
+        if not username or not password:
+            return {"message": "Required fields are missing in JSON data"}, 400
+
+        result = soap_client.service.auth_login(
+            {"username": username, "password": password}
+        )
+
+        if result.auth is not None:
+            token = result.auth.token
+            return {"message": "Login successful", "token": token}, 200
+        else:
+            return {"message": "Invalid credentials"}, 401
```

Comment:
According to [your spec](https://github.com/hawks-atlanta/proxy-python/blob/dev/docs/spec.openapi.yaml), the `message` field should be `msg` and the `succeed` one is missing. Please update either the spec or the endpoint.

Actually, I think the succeed field is redundant. Since the status code isn't in the 20X range, it can be inferred that the request didn't succeed.